src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,27 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { UserInfo } from '../UserInfo';
+import usersFromServer from '../../api/users';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const user = [...usersFromServer].find(
```

Comment:
```suggestion
  const user = usersFromServer.find(
```
it's redundant, the find method doesn't mutate the array